casepro/backend/junebug.py

```diff
@@ -40,6 +46,74 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
+        params = {}
+        for key, value in kwargs.iteritems():
+            params[key] = value
+
+        identities = self.get_paginated_response(
+            url, params=params)
+
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
```

Comment:
This is perhaps worth a comment about optouts, since otherwise people will wonder what this `name is not "removed"` bit is all about.